htdocs/accountancy/journal/sellsjournal.php

```diff
@@ -412,8 +412,13 @@
 					$bookkeeping->fk_docdet = 0; // Useless, can be several lines that are source of this record to add
 					$bookkeeping->thirdparty_code = $companystatic->code_client;
 
-					$bookkeeping->subledger_account = '';
-					$bookkeeping->subledger_label = '';
+					if ($k == $conf->global->ACCOUNTING_ACCOUNT_CUSTOMER_DEPOSIT) {
```

Comment:
You must check also that $conf->global->ACCOUNTING_ACCOUNT_CUSTOMER_DEPOSIT is defined for the if for php8 compatibility

Reply:
Thanks @eldy Resolve